lib/customstates.rb

```diff
@@ -1,50 +1,47 @@
+# frozen_string_literal: true
+
 # See `doc/THEMES.md` for more explanation of this file
 # This example adds a "transferred" state to requests.
 
 module InfoRequestCustomStates
-
-    def self.included(base)
-        base.extend(ClassMethods)
+  def self.included(base)
+    base.extend(ClassMethods)
+  end
+
+  # Work out what the situation of the request is. In addition to
+  # values of self.described_state, in base Alaveteli can return
+  # these (calculated) values:
+  #   waiting_classification
+  #   waiting_response_overdue
+  #   waiting_response_very_overdue
+  def theme_calculate_status
+    # just fall back to the core calculation
+    base_calculate_status
+  end
+
+  # Mixin methods for InfoRequest
+  module ClassMethods
+    def theme_display_status(status)
+      raise _('unknown status ') + status unless status == 'transferred'
+
+      _('Transferred.')
     end
```

Comment:
the linter has taken the existing code and simplified it. I don't know if the author intended this but this is what the existing code did. It only ever allows "transferred" as a status

Reply:
This appears to be defining an alternative state, other states are working as expected.